README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Simply, it's a way of organizing the webpage's structure in a meaningful way. It helps the person writing the code, anyone who might work with the written code later, and search engines are better able to understand the page structure.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+When an element is displayed 'inline' then it will be next to the other element on the page rather than having a break in the page. Display 'block' also allows for editing using the box model since they are standalone elements.
+
 3. What are the 4 areas of the box model?
 
+Content, Padding, Border, Margin 
+
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
 
+The cross-axis
+
 5. Explain why git is valuable to a team of developers.
 
+Git allows a team to work together by "branching" a version of the project, making changes, and then show the others on the team the changes before committing them to the overall project. Additionally, the commit option allows team members to test various versions while keepign everything organized.
```

Comment:
Awesome answer! Very true! It's amazing how closely you can work with someone who may be across the country or even across the world on a platform such as GitHub!